charm/src/charm.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Flask Charm entrypoint."""
+
+import logging
+import typing
+
+import ops
+import xiilib.django
+
+logger = logging.getLogger(__name__)
+
+
+class DjangoCharm(xiilib.django.Charm):
+    """Django Charm service."""
+
+    def __init__(self, *args: typing.Any) -> None:
+        """Initialize the instance.
+
+        Args:
+            args: passthrough to CharmBase.
+        """
+        # pylint: disable=useless-parent-delegation
```

Comment:
What is the reason for disabling this please?

Reply:
This code is the one autogenerated from 12 factor. I just added the disable line for the linter to pass (without removing the `__init__` autogenerated method).